draft-ietf-ppm-dap.md

```diff
@@ -1780,18 +1758,9 @@ This message consists of:
 
 * `part_batch_selector`: The "partial batch selector" used by the Aggregators
   to determine how to aggregate each report. Its contents depends on the
-  indicated batch mode: for time-interval mode, the `config` field is empty
-  ({{time-interval-batch-mode}}); for leader-selected tasks, the Leader
-  specifies a "batch ID" that determines the batch to which each report for
-  this aggregation job belongs ({{leader-selected-batch-mode}}).
-
-  Documents that define batch modes MUST specify the content of this field;
-  see {{extending-this-doc}} for details.
-
-  The indicated batch mode MUST match the task's batch mode. Otherwise, the
-  Helper MUST abort with error `invalidMessage`.
+  indicated batch mode. See {{batch-modes}}.
```

Comment:
brevity/clarity: let {{batch-modes}} discuss batch modes and avoid duplicating MUSTs.

The second MUST is moved to the section below